### DISCUSSION TITLE
1.0.0.62

### DISCUSSION THREAD

**Discussion by farag2** [Sophia-Community/SophiApp#69] [Announcements]
# Before running SophiApp please read [the system requirements](https://github.com/Sophia-Community/SophiApp#system-requirements)

***

<a href="https://yoomoney.ru/to/4100116615568835"><img src="https://yoomoney.ru/i/shop/iomoney_logo_color_example.png" width=220px height=46px></a>

***

[![actions](https://img.shields.io/badge/Sophia%20News-Telegram-blue?style=flat&logo=Telegram)](https://t.me/SophiaNews) [![actions](https://img.shields.io/badge/Sophia%20Chat-Telegram-blue?style=flat&logo=Telegram)](https://t.me/Sophia_Chat) [![actions](https://img.shields.io/github/workflow/status/Sophia-Community/SophiApp/Build%20Release?label=GitHub%20Actions&logo=GitHub)](https://github.com/Sophia-Community/SophiApp/actions)

![Image](https://raw.githubusercontent.com/Sophia-Community/SophiApp/master/img/0.gif)

## 1.0.0.62 — 27.05.2022

Diff from v1.0.0.60
[1.0.0.60...1.0.0.62](https://github.com/Sophia-Community/SophiApp/compare/1.0.0.60...1.0.0.62)

* Fixed bug when the app didn't launch on Windows 11 23H2 Insider Preview builds;
  * Reported by `AngelNeural`.
 * Added a new function to configure Start layout;
   * Works only on Windows 11 22H2 22509 build+ (23H2, too).

SophiApp.zip SHA256 hash ([the Compress Files step](https://github.com/Sophia-Community/SophiApp/runs/6490614058?check_suite_focus=true#step:9:16)): `87E821A432D01C35A8A9582E69250D3A7DD8D39DAF56DA9D843AAA9021848F7F`

<hr /><em>This discussion was created from the release <a href='https://github.com/Sophia-Community/SophiApp/releases/tag/1.0.0.62'>1.0.0.62</a>.</em>

**Comment by ClickToConfirm** [comment]
Suggestion(s) for the next version of the app.
- Registry tweaks; such as speeding up shutdown, disabling things that are not needed etc. 
- Services tweaks; similar to Black Viper but better.

**Comment by farag2** [comment]
Which kind of registry tweaks, speeding up shutdown? What? :D
Which things in Windows are not necessary? Never heard about them before this day. :D

Services? There's no service to disable in Windows except one, diagtrack. Bilut I need to add that disabling this service breaks opening Xbox archievments.

Don't mention Black Viper here: he's no understanding what he does with his tool. :D

**Reply by ClickToConfirm** [reply]
Reg tweaks, such as speedup shutdown, enabling browser switching, disabling services in one hit such as Hyper-v and remote connections Stuff like that.

**Reply by farag2** [reply]
> disabling services in one hit such as Hyper-v

Why user who enabled this service needs to to disable it? It doesn't exist by default. So I cannot promise to implement any of your suggestions. They makes no sense. And I do not make any harm t users' OSes. So please learn what Windows is before destroying it shitty scripts like Black Viper and similar ones.

**Reply by ClickToConfirm** [reply]
FYI I do not use BV scripts anymore for a long time. Hyper-V as I do not use them but upon installation, it's there by default. My understanding is Hyper-V is windows inbuilt virtual machine, similar to VMWare. Hence the disabling of the question of the Hyper-V service(s).  

Also with the script, when you check the checkbox that means disable/remove yeah?

**Reply by farag2** [reply]
> Hyper-V as I do not use them but upon installation, it's there by default.

Yes, it's there by default. And as the rest of services is in the manual mode. There is no need to disable any of service in Windows...

> Hyper-V is windows inbuilt virtual machine

Yes, you're right and disabled by default. You need to enable it to make its services (and features) visible.

> similar to VMWare

Actually, no but anyway. :)

>  when you check the checkbox

Screenshot? well, by default checking checkbox means that soemthing will be enabled (depends of course of the wording).

**Reply by ClickToConfirm** [reply]
"Screenshot? well, by default checking the checkbox means that something will be enabled (depends of course on the wording)."

Ahh, I get it now...appreciate it boss

**Reply by farag2** [reply]
No problem. I'm a little bit tired at work. ))) Really, my advice: just work on Windows without trying to spee it up or disabling services. IT HAS NO EFFECT AT ALL. Configure it wisely and carefully, referring with official MS documentation.

**Reply by ClickToConfirm** [reply]
Cool...keep up the good work on the script bro.
